src/components/Login.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from 'react';
+import { getUsers } from '../api';
+import { useContext } from 'react';
+import userContext from '../contexts/User';
+
+const Login = () => {
+  const [users, setUsers] = useState([]);
+
+  const { setUser } = useContext(userContext);
+
+  useEffect(() => {
+    getUsers().then(fetchedUsers => {
+      setUsers(fetchedUsers);
+    });
+  }, []);
+
+  const handleClick = event => {
+    event.preventDefault();
+    setUser(event.target.innerText);
+  };
+
+  return (
+    <div className="users">
+      {users.map(user => {
+        return (
+          <button onClick={handleClick} key={user.username}>
```

Comment:
‘Logged in as’ does not display a user name or change when the photo itself is clicked, only when the box itself is (which can result in user issues and not being able to log in)